apps/portal/src/utils/helpers.js

```diff
@@ -450,11 +450,8 @@ export function getFreeProductBenefits({site}) {
 }
 
 export function getFreeTierTitle({site}) {
-    if (hasOnlyFreeProduct({site})) {
-        return 'Free membership';
-    } else {
-        return 'Free';
-    }
+    const freeProduct = getFreeProduct({site});
+    return freeProduct?.name || 'Free';
```

Comment:
Think we should keep this backwards compatible by also returning `Free membership` if there is only one free product, otherwise the title will change before the feature is released

Reply:
Updated in [8891e61](https://github.com/TryGhost/Ghost/pull/19229/commits/8891e61097c710530e688189f547255ef41db1a7)